osm2nx.py

```diff
@@ -0,0 +1,255 @@
+"""

```

Comment:
Did you have to modify this code at all from https://github.com/brooksandrew/postman_problems_examples/blob/master/osm2nx.py ?

Reply:
No.